docs/changes.rst

```diff
@@ -24,6 +24,20 @@ Released: not yet
 
 **Incompatible changes:**
 
+* Removed Python 2.6 support.
+
+  The Python Software Foundation has stopped supporting Python 2.6 with the
+  2.6.9 release in october 2013. Since then, many Python package projects have
+  continued releasing versions for Python 2.6, and so has pywbem. In 2017 and
```

Comment:
Change: "2.6 and so has pywbem" to "2.6 including pywbem"

Reply:
done